chrome/content/zotero/zoteroPane.js

```diff
@@ -515,6 +515,18 @@ var ZoteroPane = new function()
 					this.highlightTimer = null;
 				}
 				ZoteroPane_Local.collectionsView.setHighlightedRows();
+				return;
+			} else if (event.keyCode == event.DOM_VK_BACK_QUOTE) {
```

Comment:
Oh, before I forget, one of the requests was to add read/unread toggles to context menus for item and collection trees.